src/env.cc

```diff
@@ -595,6 +595,20 @@ void Environment::AssignToContext(Local<v8::Context> context,
   TrackContext(context);
 }
 
+void Environment::UnassignToContext(Local<v8::Context> context) {
```

Comment:
Nit: The naming here is a bit awkward. `Unassign` and `To` aren't words that typically go together.

Reply:
@jasnell I don't have a good alternative that is contradictory to the name `AssignToContext`. Maybe we can rename this pair together like `RegisterContext`/`UnregisterContext`. What do you think?